packages/components/src/draggable/style.scss

```diff
@@ -11,10 +11,32 @@ body.is-dragging-components-draggable {
 }
 
 .components-draggable__clone {
+	margin-top: -$block-toolbar-height; // This puts it closer to the drag handle.
 	position: fixed;
 	padding: 20px;
 	background: transparent;
 	pointer-events: none;
 	z-index: z-index(".components-draggable__clone");
-	opacity: 0.8;
+
+	> * {
+		opacity: 0.8;
+	}
+
+	&::before {
+		content: "";
+		display: block;
+		position: absolute;
+		top: 0;
+		width: $button-size * 2;
+		height: $block-toolbar-height;
+		border-radius: $radius-block-ui;
+		background-color: $dark-gray-primary;
+		border: $border-width solid $dark-gray-primary;
+
+		// This should be reconsidered if successful.
+		background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCI+PHBhdGggZD0iTTcgMTYuNWgxMFYxNUg3djEuNXptMC05VjloMTBWNy41SDd6IiBmaWxsPSIjZmZmIi8+PC9zdmc+);
+		background-size: $icon-size;
+		background-repeat: no-repeat;
+		background-position: center center;
```

Comment:
What is this style for? Asking cause "Draggable" is supposed to be style-less as it can be used for any drag and drop interaction.

Reply:
This is what creates that black drag handle that appears when you're actually dragging. It's being painted as a pseudo element, currently, but it would arguably bet better if it were an inline element that sat in context of the draggable-clone. That might also address your point of feedback?

Reply:
I think we should move it to the block toolbar styles directly because this is very specific to that use-case.

Reply:
Moved in https://github.com/WordPress/gutenberg/pull/22673/commits/9681f280a0e1bb512804b5c63634c46c1c38a87d, let me know if that looks right!